src/hip/hip_build_utils.cpp

```diff
@@ -30,6 +30,7 @@
 #include <miopen/exec_utils.hpp>
 #include <miopen/logger.hpp>
 #include <miopen/env.hpp>
+#include <miopen/stringutils.hpp>
 #include <miopen/solver/implicitgemm_util.hpp>
```

Comment:
[Recommendation] Keep sorted.
```suggestion
#include <miopen/solver/implicitgemm_util.hpp>
#include <miopen/stringutils.hpp>
```

Reply:
No. We must turn clang-format automatic sorting on and forget about such comments.

Reply:
@CAHEK7 Enabling this right here would result in too many unrelated changes. Let's do this separately, in some future PR. Proper manual sorting in advance prevents possibility of issues there. Agree?

Reply:
@CAHEK7 Anyway this is just a recommendation that does not require explicit feedback.